src/app/pages/information/information.page.scss

```diff
@@ -0,0 +1,7 @@
+.slide-card {
+  width: 100%;
+}
+
+.multiline {
+  white-space: pre-wrap;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6012707)